islutils/pet_wrapper.cc

```diff
@@ -167,4 +167,8 @@ pet_stmt *Scop::stmt(isl::id id) const {
   return nullptr;
 }
 
+IslCopyRefWrapper<isl::schedule> Scop::schedule() {
```

Comment:
This looks like code I have written.  How about keeping it in a separate commit?  This makes clear who wrote what and hence who's to blame when it breaks.  It also avoids painful rebase management when I attempt to merge one of the branches where I do have a separate commit adding exactly the same code (44d11724a50ef669bc9e0c13aaf5615894b8c535 
63cb7b8 )

Reply:
Yes, it is your code. Can we integrate it into the master?

Reply:
Just don't meld it into your commit,  [cherry-pick](https://git-scm.com/docs/git-cherry-pick) the original commits to your branch if you need them.